syntax = "proto2";
import "game_event.proto";

message Event_AttachCard {
    extend GameEvent {
        optional Event_AttachCard ext = 2012;
    }
    optional string start_zone = 1;
    optional sint32 card_id = 2;
    optional sint32 target_player_id = 3;
    optional string target_zone = 4;
    optional sint32 target_card_id = 5;
}
